### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by fmyzjs**
add 'EDXAPP_LANGUAGE_CODE' and default it to 'en'
add 'LANGUAGE_CODE' and default it to   $EDXAPP_LANG

### CODE REVIEW

**Comment by fmyzjs** [comment]
https://github.com/edx/configuration/pull/891

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -339,6 +339,8 @@ generic_env_config:  &edxapp_generic_env
   ANALYTICS_SERVER_URL:  $EDXAPP_ANALYTICS_SERVER_URL
   FEEDBACK_SUBMISSION_EMAIL: $EDXAPP_FEEDBACK_SUBMISSION_EMAIL
   TIME_ZONE: $EDXAPP_TIME_ZONE
+  LANGUAGE_CODE : $EDXAPP_LANG
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml]: Not Quite. 
You want:
`LANGUAGE_CODE: $EDXAPP_LANGUAGE_CODE` here

And then outside this hash, where `EDXAPP_LOG_LEVEL` is defined create a new var called `EDXAPP_LANGUAGE_CODE = 'en'`

**Comment by fmyzjs** [comment]
@feanil  i commited based on https://github.com/edx/configuration/pull/891

**Comment by feanil** [comment]
Yes but what you have done here is also incorrect.  EDXAPP_LANGUAGE_CODE is not a variable that the platform knows about.  The platform config is called 'LANGUAGE_CODE'.  The EDXAPP_LANGUAGE_CODE variable is meant to let you override the default.

**Comment by fmyzjs** [comment]
@feanil So i need to define a new var called "EDXAPP_LANGUAGE_CODE "    and default to "en"
then add "LANGUAGE_CODE: $EDXAPP_LANGUAGE_CODE " ?

**Comment by feanil** [comment]
Yes.

**Comment by fmyzjs** [comment]
@feanil OK I got it

**Comment by nedbat** [comment]
@fmyzjs Have you signed a contributor's agreement? Details are at https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst

**Comment by fmyzjs** [comment]
@nedbat i have signed the individual contributor agreement and sent it

**Pull request merged**